### PR TITLE
lanraragi v0.7.0 (new formula)

### PR DESCRIPTION
**Pull request by Difegue**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/lanraragi.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
+require "language/node"
+
+class Lanraragi < Formula
+  desc "Web application for archival and reading of manga/doujinshi"
+  homepage "https://github.com/Difegue/LANraragi"
+  url "https://github.com/Difegue/LANraragi.git",
```
Comment (SMillerDev) [inline Formula/lanraragi.rb:6]: > Formulae in the core repository must have a stable version tagged by the upstream project.

https://docs.brew.sh/Acceptable-Formulae#stable-versions

File: Formula/lanraragi.rb [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  end
+
+  def install
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+    ENV.prepend_path "PERL5LIB", libexec/"lib"
+    ENV["CFLAGS"] = "-I"+libexec/"include"
+
+    resource("Image::Magick").stage do
+      inreplace "Makefile.PL" do |s|
+        s.gsub! "/usr/local/include/ImageMagick-6", "#{HOMEBREW_PREFIX}/opt/imagemagick@6/include/ImageMagick-6"
```
Comment (SMillerDev) [inline Formula/lanraragi.rb:46]:
```suggestion
        s.gsub! "/usr/local/include/ImageMagick-6", "#{Formula["imagemagick@6"].opt_include}/ImageMagick-6"
```

**Review by chenrui333 — DISMISSED** [review]
lgtm

**Comment by chenrui333** [comment]
@Difegue can you squash the commits into single one with message as `lanraragi 0.7.0 (new formula)`? Thanks!

**Comment by Difegue** [comment]
Sure! Should be good now. 😀

**Review by chenrui333 — APPROVED** [review]
💯

**Comment by chenrui333** [comment]
@Difegue, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**